custom-connectors/CopilotForFinanceCommunications/apiDefinition.swagger.json

```diff
@@ -1,765 +1,1376 @@
+          },
+          {
+            "in": "query",
+            "name": "skip",
+            "type": "integer",
+            "format": "int32"
+          },
+          {
+            "in": "query",
+            "name": "filter",
+            "description": "Format for using the 'filter' is as follows: filter=contains(fieldName,'substring') or contains(fieldName2,'substring2) or filter=contains(fieldName,'substring') and contains(fieldName2,'substring2). Applicable for all fields.",
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "range",
+            "description": "Format: range=fieldName:'value_min':'value_max' Dates are in UTC.\nApplicable to fields of type date or amount.",
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "orderby",
+            "description": "Format: orderby=fieldName:asc or orderby=fieldName:desc. Applicable for all fields.",
+            "type": "string"
+          }
+        ],
         "responses": {
-          "201": {
-            "description": "Success.",
+          "200": {
+            "description": "The request has succeeded.",
             "schema": {
+              "required": [
+                "data"
+              ],
               "type": "object",
               "properties": {
-                "chatThread": {
-                  "type": "object",
-                  "properties": {
-                    "id": {
-                      "type": "string",
-                      "description": "Chat Thread ID",
-                      "x-ms-summary": "ID"
-                    },
-                    "topic": {
-                      "type": "string",
-                      "description": "Chat Thread Topic",
-                      "x-ms-summary": "Topic"
-                    },
-                    "createdOn": {
-                      "type": "string",
-                      "format": "date-time",
-                      "description": "Chat Creation Time",
-                      "x-ms-summary": "Created On"
-                    },
-                    "createdByCommunicationIdentifier": {
-                      "$ref": "#/definitions/CommunicationIdentifier"
-                    }
+                "data": {
+                  "type": "array",
+                  "items": {
+                    "$ref": "#/definitions/CustomerActivity"
                   }
                 }
               }
             }
           },
-          "401": {
-            "description": "Unauthorized"
-          },
-          "403": {
-            "description": "Forbidden"
+          "default": {
+            "description": "An unexpected error response.",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
           }
-        },
-        "summary": "Create chat thread",
-        "description": "Create a chat thread.",
-        "operationId": "CreateChat",
```

Comment:
The operation has been deleted. This change breaks existing flows. ValuePath: operation/CreateChat

Reply:
We are adding a new template for a custom connector, so nothing is actually deleted.
Endpoint is deleted compared to prior iteration because prior iteration had incorrect swagger included.